src/constants/dictionary.js

```diff
@@ -803,7 +803,7 @@ const dictionary = {
     en: 'Identifiant',
   },
   codeUnicity: {
-    fr: 'Tous les codes doivent être uniques',
+    fr: 'Toutes les codes doivent être uniques',
```

Comment:
Toutes ?

Reply:
My previous remark was on "Tous les clarifications doivent être uniques'". 
But here, you should write "Tous".